src/facet/simulation/viz/_draw.py

```diff
@@ -90,9 +90,9 @@ def _draw(self, data: UnivariateSimulationResult) -> None:
             Sequence[Any],
             Sequence[int],
         ] = (
-            data.outputs_median().to_list(),
-            data.outputs_lower_bound().to_list(),
-            data.outputs_upper_bound().to_list(),
+            data.mean.to_list(),
+            data.lower_bound.to_list(),
+            data.upper_bound.to_list(),
             data.partitioner.partitions_,
             data.partitioner.frequencies_,
         )
```

Comment:
Nitpick: maybe it would be better to have `dict` here or `namedtuple` or `dataclass`? So access would be by name with little boilerplate, e.g.:

with `namedtuple`:
```python
from collections import namedtuple

SimulationResult = namedtuple('SimulationResult', ['mean', 'lb', 'ub', 'part', 'freq'])
sim_result = SimulationResult(data.mean.to_list(), ...)
...
self.style.draw_uplift(..., outputs_median=sim_result.mean, ...)
```

with `dataclass` with types:
```python
from dataclasses import dataclass

@dataclass
class SimulationResult:
    mean: Sequence[float]
    lb: Sequence[float]
    ...

sim_result = SimulationResult(data.mean.to_list(), ...)
```

Reply:
Thanks @mtsokol, I agree this is not ideal. I found another way to address this: class `UnivariateSimulationResult` now stores all results in a data frame, which greatly simplifies the handling and manipulation of these results in the `SimulationDrawer`. Could you please check the two new commits and re-approve the PR? Thanks!

Reply:
Looks good to me! (especially current version for sorting categories by mean), approved!